### ISSUE TITLE
( Multi-scale Deep Learning Architectures for Person Re-identification) doesn't have verification loss . .did i miss something here or the model is implemented without verification loss 

### ISSUE THREAD

**Issue by bmiftah** [KaiyangZhou/deep-person-reid#238]

**Comment by KaiyangZhou** [comment]
it does have in the paper, but is not implemented here

**Comment by bmiftah** [comment]
Thank you for the swift reply .. I think the paper has verification sub-net in the design and also it is mention in section 3.3 it discuss the verification loss , the model was also trained on verification loss from two soft-max output [](https://arxiv.org/abs/1709.05165) ... I wonder how i can include if the verification loss , I am considering using your library somehow

**Comment by bmiftah** [comment]
Being two branch model , I think the paper has verification loss in addition to classification loss , it is discussed in section 3.3 verification sub-net  [.https://arxiv.org/pdf/1709.05165.pdf.](here)  I was wondering how to include it , any suggestion where to include such loss ... I saw two loss are part of the torchreid library ..softmax and triplet  - both seems to be for multi-class classification
